src/internal/packager/git/push.go

```diff
@@ -135,7 +142,7 @@ func (g *Git) push(repo *git.Repository, spinner *message.Spinner) error {
 	} else if errors.Is(err, git.NoErrAlreadyUpToDate) {
 		message.Debugf("Repo already up-to-date, skipping fetch...")
 	} else if err != nil {
-		message.Warnf("unable to fetch remote cleanly prior to push: %w", err)
+		message.Warnf("unable to fetch remote cleanly prior to push: %v", err)
```

Comment:
Why change this from a %w?

Reply:
I think we should discuss using warns here now since we are bubbling all errors. I'd prefer to see these as `fmt.Errorf()` with `%w` (for error unwrapping) now.

Reply:
@jeff-mccoy I'm a little confused by your comment, can you say that again?

> we should discuss using warns here now

and 

> prefer to see this as `fmt.Errorf()` 

seem to contradict each other.

Reply:
Are you saying you don't want this to be a warning and you would rather just return the error immediately?

Reply:
Haha yes English is hard apparently 👀